src/style.css

```
@@ -0,0 +1,104 @@
+body {
+  margin: 0;
+  font-family: Roboto, sans-serif;
+  background-color: grey;
+}
+
+.card__code,
+.card__description,
+.card__feedback,
+.card__price,
+.card__number {
+  margin: 0;
+}
+
+.card {
+  height: 403px;
+  width: 200px;
+  position: relative;
+  box-sizing: border-box;
+  background-color: #fff;
+  padding: 31px 15px 15px;
+}
+
+.card__images {
+  width: 160px;
+  height: 134px;
+  margin: 0 5px;
```

зачем нам обнулять верхний марджин?
```suggestion
  margin-right: 5px;
```